src/unittest/python/test_client.py

```diff
@@ -0,0 +1,95 @@
+#   Copyright 2017 Covata Limited or its affiliates
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+import pytest
+import uuid
+
+from covata.delta import Client
+from covata.delta import ApiClient
+
+
+@pytest.fixture(scope="function")
+def client(api_client, key_store):
+    return Client(dict(api_client=api_client,
+                       key_store=key_store))
+
+
+@pytest.fixture(scope="function")
+def mock_signer(mocker, api_client):
+    return mocker.patch.object(api_client, "signer", return_value=mocker.Mock())
+
+
+@pytest.fixture(scope="function")
+def api_client(key_store):
+    return ApiClient(key_store)
+
+
+def test_create_identity(mocker, client, key_store, private_key, key2bytes):
+    expected_id = str(uuid.uuid4())
+
+    mocker.patch('covata.delta.crypto.generate_private_key',
+                 return_value=private_key)
+
+    mocker.patch('covata.delta.ApiClient.register_identity',
```

Comment:
since our api_client is not an object, we can use `mocker.patch.object(api_client, 'register_identity')` directly.